### ISSUE TITLE
multiqc

### ISSUE THREAD

**Issue by LisaHollstein** [MHH-RCUG/nf_wochenende#75]
@colindaven 

The multiqc process is not working right now, because 
1. the input wochenende.out.collect doesn't work, since we have to clarify wich wochenende output we need
2. it uses another conda env, that I do not have and I do not know where to get it

**Comment by colindaven** [comment]
Yes, I know, it's not complete. It needs to be redesigned, and needs a whole lot of input. Perhaps it's best as part of the wochenende process or as part of a `bam_stats` process. It's not critical though.
